internal/controller/postgrescluster/pgbackrest_test.go

```diff
@@ -2187,6 +2188,34 @@ func TestGenerateRepoHostIntent(t *testing.T) {
 			assert.Equal(t, *sts.Spec.Template.Spec.AutomountServiceAccountToken, false)
 		}
 	})
+
+	t.Run("Replicas", func(t *testing.T) {
+		assert.Equal(t, *sts.Spec.Replicas, int32(1))
+	})
+
+	t.Run("PG instances observed, do not shutdown repo host", func(t *testing.T) {
+		cluster := &v1beta1.PostgresCluster{
+			Spec: v1beta1.PostgresClusterSpec{
+				Shutdown: initialize.Bool(true),
+			},
+		}
+		observed := &observedInstances{forCluster: []*Instance{{Pods: []*corev1.Pod{{}}}}}
+		sts, err := r.generateRepoHostIntent(cluster, "", &RepoResources{}, observed)
+		assert.NilError(t, err)
+		assert.Equal(t, *sts.Spec.Replicas, int32(1))
+	})
+
+	t.Run("PG instances observed, shutdown repo host", func(t *testing.T) {
```

Comment:
:question: Should this be "No PG instances observed...."?

Reply:
Updated to "No PG instances observed".